source/faq.txt

```diff
@@ -25,11 +25,7 @@ developed by the MongoDB team and provides a native API for Kotlin
 applications to connect to MongoDB and work with data. It is implemented
 by wrapping the :driver:`MongoDB Java driver </java-drivers/>`.
 
```

Comment:
Not related to this PR necessarily, but S: move all FAQ entries to top level, right now all but the first are nested under the first even though they are not subsections

Reply:
Weird. Corrected